test/array.jl

```diff
@@ -474,6 +474,12 @@ end
 
 @testset "generator constructor" begin
     [(x, y) for x in X(10:10:50), y in Y(0.0:0.1:1.0)]
```

Comment:
For all of these we want to test the array size, that dims are actually X and Y etc,  and where a name is passed, test that it gets set.

Reply:
Ah good point. I'll add tests tomorrow.